### DISCUSSION TITLE
multiple errors running ./vhserver install command

### DISCUSSION THREAD

**Discussion by ghost** [GameServerManagers/LinuxGSM#3538] [General]
Having replicated all steps from https://linuxgsm.com/lgsm/vhserver/ including the dependencies, I'm getting errors all over the place, but it seems to be installing regardless? is this normal?
![image](https://user-images.githubusercontent.com/69369077/130179891-9b330a91-42f6-4948-9452-79b6aafd10a9.png)
![image](https://user-images.githubusercontent.com/69369077/130179914-303f43c0-d545-489e-9200-3378f798fe6d.png)

**Comment by Claiyc** [comment]
This looks to me like your server is unable to fetch any files from github.
Are you able to ping github?
Is curl properly installed and functional?
What distro do you use?
